MPF.Core/UI/ViewModels/MainViewModel.cs

```diff
@@ -1591,7 +1591,8 @@ private bool ShouldEnableDumpingButton()
             return Drives != null
                 && Drives.Count > 0
                 && this.CurrentSystem != null
-                && this.CurrentMediaType != null;
+                && this.CurrentMediaType != null
+                && (this.CurrentMediaType != MediaType.HDDVD || this.CurrentProgram != InternalProgram.Redumper);
```

Comment:
If you're doing this, it also doesn't support Blu-ray. I'd prefer a helper method instead of hardcoding into this statement.